internal/ssl_enforcement/get/get.go

```diff
@@ -0,0 +1,47 @@
+package get
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/utils"
+)
+
+func Run(ctx context.Context, projectRefArg string, fsys afero.Fs) error {
+	// 1. Sanity checks.
+	projectRef := projectRefArg
+	{
+		if len(projectRefArg) == 0 {
+			ref, err := utils.LoadProjectRef(fsys)
+			if err != nil {
+				return err
+			}
+			projectRef = ref
+		} else if !utils.ProjectRefPattern.MatchString(projectRef) {
+			return errors.New("Invalid project ref format. Must be like `abcdefghijklmnopqrst`.")
+		}
+	}
+
+	// 2. get ssl enforcement config
+	{
+		resp, err := utils.GetSupabase().GetSslEnforcementConfigWithResponse(ctx, projectRef)
+		if err != nil {
+			return fmt.Errorf("failed to retrieve SSL enforcement config: %w", err)
+		}
+		if resp.JSON200 == nil {
+			return errors.New("failed to retrieve SSL enforcement config; received: " + string(resp.Body))
+		}
+
+		if err != nil {
+			return err
+		}
+		if resp.JSON200.CurrentConfig.Database && resp.JSON200.AppliedSuccessfully {
```

Comment:
Why would get request need to check if `AppliedSuccessfully`?

Reply:
The request is stored first (so that it can be retried internally if needed), and an attempt is made to apply it afterwards. If the latter fails (for w/e reason), this indicates to the user that the request might've partially succeeded.